src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/add-polling-source-form.types.ts

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export interface JsonFormValidators {
+    required?: boolean;
+    minLength?: number;
+    maxLength?: number;
+    pattern?: string | RegExp;
+}
+
+interface JsonFormControlOptions {
+    buttonText?: string;
+    formArrayName?: string;
+}
+
+export interface JsonFormControls {
+    name: string;
+    label: string;
+    value: any;
+    type: string;
```

Comment:
Why `type` is a string and not `ControlType`

Reply:
Fixed